test/mobile/features/fixtures/maze_runner/ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.4.40f1
+m_EditorVersion: 2018.4.34f1
```

Comment:
I didn't know if this could have a negative impact on the 2017 build, but it's built ok and the tests passed, so I guess all is ok.